packages/react-lowcode/src/codegen/tests/api/api.test.ts

```diff
@@ -59,7 +59,7 @@ describe(".api tests", () => {
         test(".add column (MUI DataTable)", async () => {
             const filePath = 'src/codegen/tests/list/files/is-datatable-test-file.txt';
             const source : SourceLineCol = {lineNumber: 12, columnNumber:17, fileName:filePath};
-            const result = await addColumn(graphqlGenTs1, new CodegenRw(), source, {property: 'testdate', entityName: 'Customer'});
+            const result = await addColumn(graphqlGenTs1, new CodegenRw(), source, {property: 'testdate', entityName: 'Customer'}, "material-ui");
```

Comment:
You should not add an extra paramater. The function `addColumn()` needs to decide inside...

Reply:
probably only forgot to change in tests. It should be implemented in method `addColumn()`